src/app/index.js

```diff
@@ -261,7 +261,7 @@ tellMeWhatOSImOn()
           }
         })
 
-        onSaveAs(() => {
```

Comment:
This is here to listen to requests from the main process to save the file.
If you need a second `save-as` handler, then you need to:
 - extract the function,
 - pass the function to `onSaveAs`, and
 - use the function later for the `document.addEventListener` `save-as` handler.

Reply:
I missed where you did exactly this.  Sorry! :)